src/h5web/visualizations/shared/hooks.ts

```diff
@@ -43,3 +44,11 @@ export function useMappedArray<T>(
 export const useDataDomain = createMemo(findDomain);
 
 export const useSupportedDomain = createMemo(getSupportedDomain);
+
+export function useFrameRendering(): void {
+  const [, setNum] = useState();
+
+  useFrame(() => {
+    setNum(Math.random());
+  });
+}
```

Comment:
A bit of trickery to trigger a re-render in React on each R3F frame. This is what we used to do before, so there's no change in performance. Another way to see it is that this hook allows us to use the camera position and zoom during renders.